module/cyphersystem.js

```diff
@@ -1028,15 +1028,15 @@ function itemRollMacroQuick (actor, itemID) {
   let pointsPaid = true;
 
   if (item.type == "skill" || item.type == "teen Skill") {
-    info = titleCase(item.type) + ". Level: " + item.data.data.skillLevel;
-    if (item.data.data.skillLevel == "Inability") modifier = -1;
-    if (item.data.data.skillLevel == "Trained") modifier = 1;
-    if (item.data.data.skillLevel == "Specialized") modifier = 2;
+    info = titleCase(item.type) + ". " + game.i18n.localize("CYPHERSYSTEM.Level") + ": " + item.data.data.skillLevel;
+    if (item.data.data.skillLevel == game.i18n.localize("CYPHERSYSTEM.Inability")) modifier = -1;
+    if (item.data.data.skillLevel == game.i18n.localize("CYPHERSYSTEM.Trained")) modifier = 1;
+    if (item.data.data.skillLevel == game.i18n.localize("CYPHERSYSTEM.Specialized")) modifier = 2;
   } else if (item.type == "power Shift") {
-    let name = "Power Shift";
-    let shifts = " Shifts";
+    let name = game.i18n.localize("CYPHERSYSTEM.PowerShifts");
+    let shifts = " " + game.i18n.localize("CYPHERSYSTEM.Shifts");
     if (item.data.data.powerShiftValue == 1) {
-      shifts = " Shift"
+      shifts = " " + game.i18n.localize("CYPHERSYSTEM.Shift")
     }
     if (actor.data.data.settings.powerShifts.powerShiftsName != 0) {
       name = actor.data.data.settings.powerShifts.powerShiftsName.slice(0, -1);
```

Comment:
This one is tricky. That only works for languages which have a plural -s, where the word stem stays the same. All other languages need a different solution.

Reply:
Let's just put something like:
`Shift(s)` problem solved!

Reply:
It’s about line 1042. With that I convert the plural of whatever you call your shifts to singular. That works for English, but not for German. For example, someone might call their Power Shifts “Divine Blessings,” which will be converted to “Divine Blessing.” In German this would be “Göttliche Segen,” the singular of which would be “Göttlicher Segen.”

Reply:
Ah, my bad have not seen you were slicing the string after.

Reply:
Not answered, as I am unsure what to do there, besides proposing a singular and plural option in the settings tab.